docs/user/get-started/ios.md

```diff
@@ -0,0 +1,105 @@
+# Adding Glean to your iOS project
+
+## Usage
+
+### Integrating with your project
+
+#### Setting up the dependency
+
+Glean can be consumed through [Carthage](https://github.com/Carthage/Carthage), a dependency manager for macOS and iOS.
+For consuming the latest version of Glean, add the following line to your `Cartfile`::
+
+```
+github "mozilla/glean" "master"
+```
+
+#### Integrating with the build system
+
+In order for the Glean SDK to generate an API for your metrics, a Python environment must be accessible at build time.
+
+**TODO: Fill in how to add it to the build system**
+
+### Adding new metrics
+
+All metrics that your project collects must be defined in a `metrics.yaml` file.
+Add this file to your project and define it as an input file for the `sdk_generator.sh` script in the `Run Script` step defined before.
+The format of that file is documented [with `glean_parser`](https://mozilla.github.io/glean_parser/metrics-yaml.html).
+To learn more, see [adding new metrics](adding-new-metrics.md).
+
+> **Important**: as stated [before](../adding-glean-to-your-project.md#before-using-glean), any new data collection requires documentation and data-review.
+> This is also required for any new metric automatically collected by the Glean SDK.
+
+### Adding custom pings
+
+Please refer to the [custom pings documentation](../pings/custom.md).
+
+> **Important**: as stated [before](../adding-glean-to-your-project.md#before-using-glean), any new data collection requires documentation and data-review.
+> This is also required for any new metric automatically collected by the Glean SDK.
+
+### Testing metrics
+
+In order to make testing metrics easier 'out of the box', all metrics include a set of test API functions in order to facilitate unit testing.  These include functions to test whether a value has been stored, and functions to retrieve the stored value for validation.  For more information, please refer to [Unit testing Glean metrics](../testing-metrics.md).
+
+### Adding metadata about your project to the pipeline
+
+In order for data to be collected from your project, metadata must be added to `probe_scraper`.
```

Comment:
This is out of scope for this bug, but do you think we should sync up this section with https://docs.telemetry.mozilla.org/concepts/glean/glean.html#how-to-use-glean ? We require people to file data-engineering bugs for setting up these things, I think.

@fbertsch, how do we want this process to be? When adding the SDK to an app, do we want people to file bugs and we take care of the rest?

Reply:
This makes the documentation very _Mozilla_ specific.  Should we be careful of this if Glean is going to be used by third-parties?

Reply:
This is a good point. However, right now, our docs are heavily *Mozilla* specific. For example, all of our docs assume we're being consumed through `mozilla.components.service.glean` (so through our A-C shim). We might want to document that in the future. I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1589392) for it.